config.go

```diff
@@ -11,15 +12,23 @@ import (
 
 type profiles map[string]map[string]string
 
-func parseProfiles() (profiles, error) {
-	file := os.Getenv("AWS_CONFIG_FILE")
+func configFile() (file string, err error) {
+	file = os.Getenv("AWS_CONFIG_FILE")
 	if file == "" {
 		usr, err := user.Current()
 		if err != nil {
-			return nil, err
+			return file, err
 		}
 		file = usr.HomeDir + "/.aws/config"
 	}
+	return file, err
+}
+
+func parseProfiles() (profiles, error) {
+	file, err := configFile()
+	if err != nil {
+		return nil, err
+	}
 
 	log.Printf("Parsing config file %s", file)
 	f, err := ini.LoadFile(file)
```

Comment:
Sorry, and this doesn't need to close cos the ini package takes care of it. I guess I'm out of nits.